### DISCUSSION TITLE
Nvidia kernel modules cannot be built because of gcc

### DISCUSSION THREAD

**Discussion by Helge102** [xanmod/kernel#441] [General]
The title says it all.

A newer version of gcc-13 has been in use in Debian Sid for a few days.

_warning: the compiler differs from the one used to build the kernel
  The kernel was built by: gcc-13 (Debian 13.3.0-1) 13.3.0
  You are using:           gcc-13 (Debian 13.3.0-3) 13.3.0_

Please rebuild the kernel with gcc-13.3.0-3.
Thanks.

**Comment by sgtcoder** [comment]
I have a similar issue with vmware kernel modules. The only work around I found was to install gcc-13 from sid. I stopped doing that because all the dependencies require certain versions and over time more and more packages convert to sid.

I believe the team had a repo that prevented this. It was some fedora url. It was in Github Issues, but that's no longer available.
